robotiq_cpp_control/include/gripper_interface.h

```diff
@@ -40,70 +40,106 @@
  *		1) C++ objects, as seen in rs_gripper_interface_test.cpp
  *		2) Publish commands on the ROS topic "grip_command".
  * @version 1.0
- * 
+ *
  * @author  Adam Allevato <adam.d.allevato@gmail.com>
  * @copyright BSD 3-paragraph
  * @date    Dec 4, 2015
  */
+namespace robotiq_cpp_control
+{
+
+namespace MODE
+{
+/**
+* Possible modes for the gripper
+*/
+enum Mode {
+    MODE_BASIC = 0,
+    MODE_PINCH,
+    MODE_WIDE,
+    MODE_SCISSOR
+};
+}
+typedef MODE::Mode Mode;
+
+
+namespace STATUS
```

Comment:
Should be `Statuses`  That is proper english, according to [here](http://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status) ;)